azure-nextgen-cs-cosmosdb-logicapp/MyStack.cs

```diff
@@ -0,0 +1,216 @@
+// Copyright 2016-2021, Pulumi Corporation.  All rights reserved.
+
+using Pulumi;
+using System.Collections.Generic;
+using Authorization = Pulumi.AzureNextGen.Authorization.Latest;
+using DocumentDB = Pulumi.AzureNextGen.DocumentDB.Latest;
+using Logic = Pulumi.AzureNextGen.Logic.Latest;
+using Resources = Pulumi.AzureNextGen.Resources.Latest;
+using Storage = Pulumi.AzureNextGen.Storage.Latest;
+using Web = Pulumi.AzureNextGen.Web.Latest;
+
+class MyStack : Stack
+{
+    public MyStack()
+    {
+        // Create an Azure Resource Group
+        var resourceGroup = new Resources.ResourceGroup("resourceGroup", new Resources.ResourceGroupArgs
+        {
+            ResourceGroupName = "logicappdemo-rg",
+        });
+
+        // Create an Azure resource (Storage Account)
+        var storageAccount = new Storage.StorageAccount("storageAccount", new Storage.StorageAccountArgs
+        {
+            AccountName = "logicappdemosa21",
```

Comment:
Nit - with 0.7.0 (autonaming) we should just let the provider handle the names here.